app/assets/stylesheets/welcome.css.scss

```diff
@@ -0,0 +1,26 @@
+#banner {
+	background-image: url(image_path('welcome.jpg'));
+	background-size: cover;
```

Comment:
Line should be indented with spaces, not tabs<br>Properties should be ordered background-image, background-position, background-repeat, background-size, display, min-height, width